src/main/com/fulcrologic/rad/report.cljc

```diff
@@ -376,6 +376,27 @@
         (start-report! app report-class {:route-params route-params})
         (comp/transact! app [(dr/target-ready {:target report-ident})])))))
 
+(s/def ::arglist (s/and vector?
+                        (fn [[this props]]
+                          (and (not= this props)
+                               (not= this '_)
+                               (not= props '_)))))
+
+(s/def ::columns (s/coll-of symbol? :kind vector?))
+(s/def ::row-pk symbol?)
+(s/def ::source-attribute keyword?)
```

Comment:
This could be stored in a var...so could be a symbol

Reply:
As abve, just replaces

```clojure
(req! &env sym options ::source-attribute keyword?)
```